src/docs/development/platform-integration/platform-channels.md

```diff
@@ -632,47 +630,37 @@ and the app displays 'battery info unavailable'.
 
 If you expect to use your platform-specific code in multiple Flutter apps,
 it can be useful to separate the code into a platform plugin located
-in a directory outside your main application. See [developing
-packages](/docs/development/packages-and-plugins/developing-packages)
-for details.
+in a directory outside your main application.
+See [developing packages][] for details.
 
 ## Publish platform-specific code as a package {#publish}
 
-To share your platform-specific code with other developers in the Flutter
-ecosystem, see [publishing
-packages](/docs/development/packages-and-plugins/developing-packages#publish).
+To share your platform-specific code with other developers
+in the Flutter ecosystem, see [publishing packages][].
 
 ## Custom channels and codecs
```

Comment:
There's an example of a custom codec in the cloud_firestore plugin, that is able to serialize/deserialize many more types than the default ones, here:

https://github.com/FirebaseExtended/flutterfire/blob/master/packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/method_channel/utils/firestore_message_codec.dart

Reply:
Excellent!